### DISCUSSION TITLE
Raspberry Pi + Touch Screen Support

### DISCUSSION THREAD

**Discussion by RK11111111111** [monocasual/giada#715] [General]
Does Giada support Rasperry Pi especially touch.  I want to try a looper, but I thought I would ask before I get into it.  I see it supports ALSA, which is good on raspberry pi.

I plan on using it in conjunction with an audio interface and a midi controller.  I have them mapped on raspian OS for using Reaper right now.

Ideally, I would be able to boot the OS directly to the looper so in a live situation I don't have to mess with the desktop.  So, using the sway desktop manager Sway instead of X.  I've done this successfully with Mixxx and Surge XT for a DJ mixer and synth.

I thought I would ask before diving in.  I've found the touch interface to be very poorly supported in Linux.  However, surge XT has a touch mode.  For example, accessing right click menus is a pain.  It is still an issue with Mixxx, but they have a flexible meta language for redesigning layouts, and I was able to design around the issue.

**Comment by gvnnz** [comment]
Hi @RK11111111111, this is a good question and the answer is: maybe? If my memory serves, someone was able to successfully run Giada on Rasperry Pi and other similar devices. We don't officially support it, meaning we don't provide builds specifically tailored for that. It would be great if you could try it and share your findings!
